openmc/mgxs/mgxs.py

```diff
@@ -1806,8 +1819,6 @@ def get_xs(self, in_groups='all', out_groups='all',
                                           filter_bins=filter_bins,
                                           nuclides=query_nuclides, value=value)
 
-        xs = np.nan_to_num(xs)
```

Comment:
I just moved in down to after where the `micro` could be computed. I went through all the `get_xs(...)` methods and tried to make them all consistent with where they do nan to num and squeeze the array.